Pipfile

```diff
@@ -41,13 +42,13 @@ pretty = '''
             --remove-all-unused-imports --remove-unused-variables \
             wkconnect benchmarks stubs && \
         black wkconnect benchmarks stubs && \
-        isort --recursive wkconnect benchmarks stubs \
+        isort --recursive wkconnect benchmarks \
```

Comment:
`isort` and `black` have incompatible opinions about `*.pyi` stubs, therefore disabling isort here.